scripts/create-package.js

```diff
@@ -112,13 +112,14 @@ const packageJSON = `
 
 const tsConfig = `
 {
-  "extends": "../../package.tsconfig.json",
+  "extends": "../../tools/config/package.tsconfig.json",
```

Comment:
what about in cases where this isn't a relative import

Reply:
updated